sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubsException.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 using System;
+using System.ComponentModel;
```

Comment:
Just wondering why this was added?

Reply:
Good catch.  That got added when I considered adding the decorating attribute to hide from Intellisense and apparently forgot to remove and sort.   I'll zap that.